PeerioTechnologies/peerio-desktop
# fix: ContactList cutoff on small window sizes

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/5712/desktop-in-contacts-on-small-window-sizes-the-icons-next-to-contact-entry-get-cut-off

#### Testing instructions  
While in ContactList, make the Peerio window very small
* the buttons on the right should no longer get cut off (compare to dev/prod or see screenshot in Clubhouse)
* if a user has a very long fullname, it should properly ellipsize and not make the spacing weird on the avatar or the chat/star buttons

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
